plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastoreModule.java

```diff
@@ -50,10 +50,6 @@ protected void setup(Binder binder)
             bindMetastoreModule("thrift", new ThriftMetastoreModule());
             bindMetastoreModule("file", new FileMetastoreModule());
             bindMetastoreModule("glue", new GlueMetastoreModule());
-            // Load Alluxio metastore support through reflection. This makes Alluxio effectively an optional dependency
-            // and allows deploying Trino without the Alluxio jar. Can be useful if the integration is unused and is flagged
-            // by a security scanner.
-            bindMetastoreModule("alluxio-deprecated", deferredModule("io.trino.plugin.hive.metastore.alluxio.AlluxioMetastoreModule"));
```

Comment:
You can also remove the `deferredModule` method, it was added specifically for Alluxio and I don't hing we'll need it any more.